spinalcordtoolbox/utils/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .fs import *
-from .sys import *
-from .sys import __version__, __sct_dir__, __data_dir__, __deepseg_dir__
-from .shell import *
+from .fs import *  # noqa: F401, F403
+from .sys import *  # noqa: F401, F403
+from .sys import __version__, __sct_dir__, __data_dir__, __deepseg_dir__  # noqa: F401
+from .shell import *  # noqa: F401, F403
```

Comment:
I was curious if there was an alternate pattern we could use here that doesn't require the use of `noqa` comments, since I figure if `noqa` comments are necessary, then perhaps we're doing something in a non-standard way?

The closest I got to a useful suggestion was [proposing the use of `__all__`](https://stackoverflow.com/a/31079085) to define the public API for a module. But, `__all__` seems intended for trying to [distinguish between public and private interfaces](https://peps.python.org/pep-0008/#public-and-internal-interfaces) in a module? Whereas here, we're just trying to alias submodules to `import utils`.

Given that we've talked about not necessarily being perfectly Pythonic when it comes to API design (as we don't intend to expose our API to downstream developers), I think going with the `noqa` comments is the most sensible thing to do, especially given the scope of the PR.

Reply:
I thought about `__all__` as well, but (apart from any further subtleties) we don't have an explicit list to give it, since we're importing `*`. We're definitely not being pythonic here. Possibly I should just open an issue to review who imports what from where and get rid of this `__init__.py` file altogether.